nexus/posts/serializers.py

```diff
@@ -0,0 +1,26 @@
+from rest_framework import serializers
+
+from . import models
+
+
+class PostSerializer(serializers.ModelSerializer):
+    posted_by = serializers.ReadOnlyField(source='posted_by.email')
+    created_at = serializers.SerializerMethodField()
```

Comment:
These are not needed.

By default, the format is ISO 9601 for dates. You can update that for `created_at` and `modified_at` timestamps that are inherited from `TimestampedUUIDModel` from REST framework settings.

Reply:
API is giving `2018-07-31T08:39:29.828870Z` in json which is not user-friendly and it'll also be hard to provide the same format while creating a post.

Reply:
I already told you what to do. Please read my comment again.